modules/blocks/chain.js

```diff
@@ -389,22 +389,22 @@ __private.applyConfirmedStep = function(block, tx) {
 					{ publicKey: transaction.senderPublicKey },
 					(accountErr, sender) => {
 						if (accountErr) {
-							const err = `Failed to get account to apply transaction: ${
+							const err = `Failed to get account to applyConfirmed transaction: ${
```

Comment:
For consistency may be good, but semantically this statement does not feel good to me. 

> Failed to get account to applyConfirmed transaction

May be instead

> Failed to get account to apply transaction confirmed state. 

function naming convention are internal stuff, while log messages are what our app distribute outside as information. So it should be semantically and grammatically correct as well.